src/components/article/viewArticleComponents.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import Pagination from 'rc-pagination';
+import PropTypes from 'prop-types';
+import Pages from 'lodash';
+import './viewArticle.scss';
+import star from '../../../public/assets/images/images/yellowStar.png';
+import blankStar from '../../../public/assets/images/images/blankRateStar.png';
+import userImage from '../../../public/assets/images/images/userIcon.png';
+// import ArticleCard from './articleCard';
+import ArticleCard from '../common/articleCard';
+import coverImage from '../../../public/assets/images/nature.jpeg';
+
+export const paginate = (lists, pageNumber, pageSize) => {
+  const startIndex = (pageNumber - 1) * pageSize;
+  return Pages(lists).slice(startIndex).take(pageSize).value();
+};
+
+export const LeftSideBar = ({
+  firstname, lastname, profilePic, LeftSideStyles, profilePicStyles,
+}) => (
+
+  <div className="left-side">
+    <div className="icon" style={LeftSideStyles}>
+      <div style={profilePicStyles}>
```

Comment:
how about creating different files, instead of putting all things in one file

Reply:
Nice suggestion Salvi. I think we should prioritize the motivation as to why in the first place we have two separate API endpoints. i.e endpoint for getting all articles and getting a single articles. I used the logic above to separate concerns.